json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -20,7 +20,9 @@ protected function result() {
 		}
 
 		foreach ( array( 'posts', 'comments', 'users' ) as $module_name ) {
-			if ( isset( $args[ $module_name ] ) ) {
+			if ( 'users' === $module_name && isset( $args[ $module_name ] ) && 'initial' === $args[ $module_name ] ) {
```

Comment:
this special-casing is kind of messy, but I couldn't immediately think of a cleaner way to express it that wasn't overly abstracted.